### PR TITLE
Client cert http auth

### PR DESCRIPTION
**Pull request by PrarthonaPaul**

### CODE REVIEW

File: client-cert-auth-without-roles/configure-elytron-server.cli [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Configure a security domain	
+/subsystem=elytron/security-domain=clientCertDomain:add(realms=[{realm=idsRealm}], default-realm=idsRealm, pre-realm-principal-transformer=myChainedTransformer, permission-mapper=default-permission-mapper)
+
+# Configure the CLIENT_CERT HTTP mechanism on elytron
+/subsystem=elytron/configurable-http-server-mechanism-factory=configuredCert:add(http-server-mechanism-factory=global, properties={org.wildfly.security.http.skip-certificate-verification=true})
+/subsystem=elytron/http-authentication-factory=clientCertAuth:add(http-server-mechanism-factory=configuredCert, security-domain=clientCertDomain , mechanism-configurations=[{mechanism-name=CLIENT_CERT}])
+
+batch
+#configure the undertow subsystem	
+/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context, value=applicationSSC)
```
Comment (fjuma) [inline client-cert-auth-without-roles/configure-elytron-server.cli:38]: The `https-listener` from the default configuration now already references the `applicationSSC`, so this line can be removed.

File: client-cert-auth-without-roles/simple-webapp/pom.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <name>simple-webapp Maven Webapp</name>
+    <url>http://maven.apache.org</url>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <version.junit>4.13.2</version.junit>
+        <version.wildfly.maven.plugin>2.0.0.Final</version.wildfly.maven.plugin>
+        <version.server.bom>27.0.1.Final</version.server.bom>
```
Comment (fjuma) [inline client-cert-auth-without-roles/simple-webapp/pom.xml:17]: We can use 28.0.1.Final here (that's the latest WildFly version).

File: client-cert-with-authorization/configure-elytron-server.cli [hunk 3]
```diff
@@ -9,21 +9,25 @@
 
 # Create a security realm with two identities, 123456 and 456789
 /subsystem=elytron/filesystem-realm=idsRealm:add(path=ids,relative-to=jboss.server.config.dir)
-/subsystem=elytron/filesystem-realm=idsRealm:add-identity(identity=123456)
```
Comment (fjuma) [inline client-cert-with-authorization/configure-elytron-server.cli:12]: Looks like this is where the identities were accidentally being removed from the client-cert-with-authorization example.

File: client-cert-with-authorization/configure-elytron-server.cli [hunk 4]
```diff
@@ -9,21 +9,25 @@
...
 /subsystem=elytron/configurable-http-server-mechanism-factory=configuredCert:add(http-server-mechanism-factory=global, properties={org.wildfly.security.http.skip-certificate-verification=true})
 /subsystem=elytron/http-authentication-factory=clientCertAuth:add(http-server-mechanism-factory=configuredCert, security-domain=clientCertDomain, mechanism-configurations=[{mechanism-name=CLIENT_CERT}])
-/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=clientCertAuth,override-deployment-config=true)
+
+batch
+#configure the undertow subsystem	
+/subsystem=undertow/application-security-domain=other:undefine-attribute(name=security-domain)
+/subsystem=undertow/application-security-domain=other:write-attribute(name=http-authentication-factory,value=clientCertAuth)
+/subsystem=undertow/application-security-domain=other:write-attribute(name=override-deployment-config,value=true)
+run-batch
```
Comment (fjuma) [inline client-cert-with-authorization/configure-elytron-server.cli:30]: Since this PR is for the new example, we shouldn't be making any changes to the client-cert-with-authorization script here.

File: client-cert-auth-without-roles/configure-elytron-server.cli [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+#configure a idsRealm for the user to map to
+/subsystem=elytron/filesystem-realm=idsRealm:add(path=ids, relative-to=jboss.server.config.dir)
+/subsystem=elytron/filesystem-realm=idsRealm:add-identity(identity=client-cert-user)
+
+
+# configure the Server ssl context to connect it to the key manager and the trust manager
+/subsystem=elytron/server-ssl-context=applicationSSC:write-attribute(name=key-manager, value=twoWayKM)
+/subsystem=elytron/server-ssl-context=applicationSSC:write-attribute(name=trust-manager, value=twoWayTM)
+/subsystem=elytron/server-ssl-context=applicationSSC:write-attribute(name=need-client-auth, value=true)
+/subsystem=elytron/server-ssl-context=applicationSSC:write-attribute(name=want-client-auth, value=true)
```
[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/configure-elytron-server.cli:22]: When I undefined both of them, it would no longer ask me for the certificates and would not go through the authrntication process. 
Should I only set `need-client-auth` to true then since `want-client-auth` would be ignored anyway?

[the remark this answers is not in the document]
Comment (fjuma) [reply client-cert-auth-without-roles/configure-elytron-server.cli:22]: Ah, ok. Yes, let's just set `need-client-auth` to `true`.

File: client-cert-auth-without-roles/configure-elytron-server.cli (hunk 1 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/configure-elytron-server.cli:38]: Fixed!

File: client-cert-auth-without-roles/simple-webapp/pom.xml (hunk 2 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/simple-webapp/pom.xml:17]: fixed!

File: client-cert-auth-without-roles/README.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
+## Using Certificate Based Authentication Without Roles In A Mutual TLS-Connection
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:1]: Maybe we could reword this to something like:

Using Certificate Based Authentication with Mutual TLS without Roles

File: client-cert-auth-without-roles/README.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+
+  
+  
+
```
[the remark this answers is not in the document]
Comment (fjuma) [reply client-cert-auth-without-roles/README.md:5]: Same with the rest of the doc, there appear to be a lot of blank lines in between sections, typically one is enough. It doesn't affect the format when actually viewing the file but makes the raw file a bit harder to read.

File: client-cert-auth-without-roles/README.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+This example will walk you through how to establish a secure connection via mutual TLS connection on a web application deployed to WILDFLY. We will be doing so using CLIENT_CERT HTTP authentication.
+
+  
+
+CLIENT_CERT authentication, is a method where both the client and server authenticate each other using their digital certificates in a TLS connection. During the TLS handshake, the client and server exchange and validate certificates, ensuring the identity of both parties. In this example we will be using certificates signed by an example CA. We will be making use of the `Dynamic-certificate` repository.
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:10]: I like that you've added an explanation here! Would be good to tweak this slightly as follows:

"CLIENT_CERT authentication, is a method where both the client and server authenticate each other using their digital certificates in a TLS connection."

should be changed to something like:

When using mutual TLS, both the client and the server will authenticate each other using their digital certificates.

Comment (fjuma) [inline client-cert-auth-without-roles/README.md:10]: [`dynamic-certificates`](https://github.com/wildfly-security-incubator/elytron-examples/tree/main/dynamic-certificates) project

File: client-cert-auth-without-roles/README.md [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+
+### Overview
+
+  
+
+* [ Certificate generation ](#certGeneration)
+
+* [ Server configuration ](#serverConfiguration)
+
+* [ Deploying and accessing the application ](#deployingApp)
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:22]: I like the way you've split this into sections!

File: client-cert-auth-without-roles/README.md [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+
+cd $PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/simple-webapp
+
+mvn clean install wildfly:deploy
+
+```
+
+  
+
+<b>Note: </b>there is also a repository under ```elytron-examples``` which contains a `simple-webapp` folder. But we will not be using that. Instead, we will be deploying a slightly modified version of that web application located inside this repository.
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:162]: s/repository/project

Comment (fjuma) [reply client-cert-auth-without-roles/README.md:162]: (s/one/two notation indicates that where you currently have "one", that should be changed to "two" instead on this line)

File: client-cert-auth-without-roles/README.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+### Overview
+
+* [ Certificate generation ](#certGeneration)
+
+* [ Server configuration ](#serverConfiguration)
+
+* [ Deploying and accessing the application ](#deployingApp)
+
+<a  name="certGeneration"></a>
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:15]: Instead of inserting this manually, I think you should be able to make use of toc::[] to insert the table of contents automatically for you with appropriate links. Take a look at:

* 2022-09-01-bearer-only-support-openid-connect.adoc in the _posts dir
* it's rendered here: https://wildfly-security.github.io/wildfly-elytron/blog/bearer-only-support-openid-connect/

File: client-cert-auth-without-roles/README.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+Finally, copy the server.keystore and server.truststore files to your WildFly server instance:
+```
+
+cp $PATH_TO_ELYTRON_EXAMPLES/dynamic-certificates/target/server.* $WILDFLY_HOME/standalone/configuration
+
+```
+<a  name="serverConfiguration"></a>
+
+### Server Configuration
+The following set of instructions are going to be used to configure the wildfly server in order to establish the mutual TLS connection. We will be deploying a simple web application under the `elytron-examples/client-cert-auth-without-roles/simple-webapp` folder.
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:77]: s/wildfly/WildFly

File: client-cert-auth-without-roles/README.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+We’re going to make use of the ```simple-webapp``` project. It can be deployed using the following commands:
+
+```
+
+cd $PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/simple-webapp
+
+mvn clean install wildfly:deploy
+
+```
+<b>Note: </b>there is also a `elytron-examples/simple-webapp` repository. But we will not be using that. Instead, we will be deploying a slightly modified version of that web application located inside this repository.
```
[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/README.md:101]: I am not quite sure I understand. 
What does the "s/" represent?

File: client-cert-auth-without-roles/README.md [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+### Server Configuration
+The following set of instructions are going to be used to configure the wildfly server in order to establish the mutual TLS connection. We will be deploying a simple web application under the `elytron-examples/client-cert-auth-without-roles/simple-webapp` folder.
+
+Please navigate to the elytron server home directory and enter the following command.
+```
+
+$SERVER_HOME/bin/jboss-cli.sh --connect --file=$PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/configure-elytron-server.cli
+
+```
+Take a look at the ```configure-elytron-server.cli``` file in the ```client-cert-auth-without-roles``` directory for more
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:85]: I think it would be good to provide some more details about what is being configured on the server side. 

As an example, take a look at the server configuration descriptions in the following posts:
* https://wildfly-security.github.io/wildfly-elytron/blog/ejb-over-http/
* https://wildfly-security.github.io/wildfly-elytron/blog/ejb-over-tls/

File: client-cert-auth-without-roles/README.md [hunk 15]
```diff
@@ -0,0 +1,154 @@
...
+
+Now try accessing the application using https://localhost:8443/simple-webapp
+
+Note that since the server's certificate won't be trusted by your browser, you'll need to manually confirm that
+this certificate is trusted or configure your browser to trust it.
+
+Select the certificate for Bob.Smith.123456 and then click on
+“Access Secured Servlet”. This will authenticate the client and will recognize that Bob accessed the web application and print "Login Successful!". 
+
+Try closing the web app and open it in a new tab. This time select Alice's certificate. Since the server is not authenticated with Alice's CLIENT_CERT, the applciation will not recognize her and will print "Login Unsuccessful.".
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:141]: Just to make sure I understand, is there really a difference between Alice and Bob in this case? Both of their certificates are signed by the example CA and both have CN values that will match on the regular expression from the regex principal transformer so I am wondering why logging in with Alice's cert is unsuccessful.

File: client-cert-auth-without-roles/README.md [hunk 16]
```diff
@@ -0,0 +1,154 @@
...
+Once you are ready to restore your server back to what it was, please enter the following on your terminal: 
+
+```
+
+$SERVER_HOME/bin/jboss-cli.sh --connect --file=$PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/restore-elytron-configuration.cli
+
+```
+
+This example has shown to secure a web application deployed to WildFly using the CLIENT_CERT HTTP authentication
+mechanism with two-way SSL with authorization. It has also demonstrated that individual client certificates do not need
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:153]: "with authorization" should be removed since we aren't using roles.

File: client-cert-with-authorization/configure-elytron-server.cli (hunk 3 above)
Comment (fjuma) [reply client-cert-with-authorization/configure-elytron-server.cli:12]: This line needs to be added back.

File: client-cert-with-authorization/configure-elytron-server.cli [hunk 17]
```diff
@@ -9,21 +9,25 @@
...
 /subsystem=elytron/filesystem-realm=idsRealm:add-identity-attribute(identity=123456,name=Roles,value=[Users])
-/subsystem=elytron/filesystem-realm=idsRealm:add-identity(identity=456789)
 
 # Configure a security domain that references our security realm
 /subsystem=elytron/security-domain=clientCertDomain:add(realms=[{realm=idsRealm}], default-realm=idsRealm, pre-realm-principal-transformer=myChainedTransformer, permission-mapper=default-permission-mapper)
 
 # Configure two-way SSL
 /subsystem=elytron/key-store=serverTS:add(path=server.truststore,relative-to=jboss.server.config.dir,credential-reference={clear-text=truststorepass},type=JKS)
-security enable-ssl-http-server --key-store-path=server.keystore --key-store-path-relative-to=jboss.server.config.dir --key-store-password=keystorepass --trust-store-name=serverTS
+security enable-ssl-http-server --key-store-path=server.keystore --key-store-path-relative-to=jboss.server.config.dir --key-store-password=keystorepass --trust-store-name=serverTS --override-ssl-context
```
Comment (fjuma) [inline client-cert-with-authorization/configure-elytron-server.cli:19]: This change shouldn't be made here since it's part of the other PR.

File: client-cert-with-authorization/configure-elytron-server.cli (hunk 4 above)
Comment (fjuma) [reply client-cert-with-authorization/configure-elytron-server.cli:30]: Same here, these changes shouldn't be part of this PR.

**Comment by fjuma** [comment]
@Skyllarr Please review as well.

File: client-cert-auth-without-roles/README.md (hunk 11 above)
Comment (fjuma) [reply client-cert-auth-without-roles/README.md:15]: Please ignore this comment :) Going back and forth between reviewing the elytron-example PR and the blog post PR and was in blog post PR review mode. This comment isn't applicable here.

File: client-cert-auth-without-roles/README.md (hunk 14 above)
Comment (fjuma) [reply client-cert-auth-without-roles/README.md:85]: Same here - please ignore this comment. The details are meant for the blog post not the README for the elytron-example so this comment isn't applicable here.

Comment (fjuma) [reply client-cert-auth-without-roles/README.md:85]: And I see you have included the details in the blog post so that's great, thanks for that!

File: client-cert-auth-without-roles/README.md (hunk 15 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/README.md:141]: In the instructions, I only added Bob's cert. So, the browser would not have Alice's cert. But if they do enter alice's cert in the browser as well, both would work.

File: client-cert-auth-without-roles/README.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/README.md:101]: nvm I see what you mean now!

File: client-cert-with-authorization/configure-elytron-server.cli (hunk 3 above)
Comment (PrarthonaPaul) [reply client-cert-with-authorization/configure-elytron-server.cli]: this is from the other repo, so I removed the changes from this PR.

File: client-cert-with-authorization/configure-elytron-server.cli [hunk 18]
```diff
@@ -9,21 +9,25 @@
 
 # Create a security realm with two identities, 123456 and 456789
 /subsystem=elytron/filesystem-realm=idsRealm:add(path=ids,relative-to=jboss.server.config.dir)
-/subsystem=elytron/filesystem-realm=idsRealm:add-identity(identity=123456)
 /subsystem=elytron/filesystem-realm=idsRealm:add-identity-attribute(identity=123456,name=Roles,value=[Users])
-/subsystem=elytron/filesystem-realm=idsRealm:add-identity(identity=456789)
```
[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply client-cert-with-authorization/configure-elytron-server.cli]: removed the changes in this file since this belongs to a different PR

File: client-cert-auth-without-roles/README.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (fjuma) [reply client-cert-auth-without-roles/README.md]: Ah, sorry, this notation is just used to indicate that the contents in the first part should be replaced by the second part. So wherever you've mentioned "repository" should be reworded to "project" instead.

File: client-cert-auth-without-roles/README.md (hunk 15 above)
Comment (fjuma) [reply client-cert-auth-without-roles/README.md:141]: Logging in with Alice's certificate would have the same behaviour as when logging in with Bob's certificate though. So you might not need to mention or make use of Alice's certificate at all.

To show an unsuccessful login, you'd need to use something like a self-signed certificate that won't be trusted by the server.

File: client-cert-auth-without-roles/simple-webapp/src/main/webapp/WEB-INF/web.xml [hunk 19]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<web-app version="2.5" xmlns="http://java.sun.com/xml/ns/javaee"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+   metadata-complete="false">
+
+    <login-config>
+        <auth-method>BASIC</auth-method>
```
Comment (Skyllarr) [inline client-cert-auth-without-roles/simple-webapp/src/main/webapp/WEB-INF/web.xml:9]: @PrarthonaPaul Just a minor, do we need a BASIC authentication method configured here?

Comment (Skyllarr) [reply client-cert-auth-without-roles/simple-webapp/src/main/webapp/WEB-INF/web.xml:9]: IIUC it can be changed to `CLIENT-CERT` ? If that is the case,  you can maybe remove the `/subsystem=undertow/application-security-domain=other:write-attribute(name=override-deployment-config,value=true)` ? Or is there a different reason this command is used in the the configure-elytron-server.cli?

Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/simple-webapp/src/main/webapp/WEB-INF/web.xml:9]: I am not too sure. I don't believe I configured it specifically. It just came with the original `simple-webapp` project. 
The only things I really changed was getting rid of roles in the .java file.

File: client-cert-auth-without-roles/simple-webapp/.gitignore [hunk 20]
```diff
@@ -0,0 +1,4 @@
+.classpath
```
Comment (Skyllarr) [inline client-cert-auth-without-roles/simple-webapp/.gitignore:1]: @PrarthonaPaul Just a minor, this file should be removed

File: client-cert-auth-without-roles/simple-webapp/pom.xml [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
```
Comment (Skyllarr) [inline client-cert-auth-without-roles/simple-webapp/pom.xml:40]: @PrarthonaPaul  Just  a total minor, junit is not used in the code so we can remove it from the dependencies

File: client-cert-auth-without-roles/README.md [hunk 22]
```diff
@@ -0,0 +1,154 @@
...
+
+Please navigate to the elytron server home directory and enter the following command.
+```
+
+$SERVER_HOME/bin/jboss-cli.sh --connect --file=$PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/configure-elytron-server.cli
+
+```
+Take a look at the `configure-elytron-server.cli` file in the `client-cert-auth-without-roles` directory for more
+
+details on the configuration.
```
Comment (Skyllarr) [inline client-cert-auth-without-roles/README.md:87]: @PrarthonaPaul Just a minor, the formatting here is off

File: client-cert-auth-without-roles/README.md [hunk 23]
```diff
@@ -0,0 +1,154 @@
...
+* Both client certificates are issued by an example certificate authority with distinguished name: `CN=Elytron CA, ST=Elytron, C=UK, EMAILADDRESS=elytron@wildfly.org, O=Root Certificate Authority`
+
+*  `server.truststore` contains this certificate authority's certificate
+
+Next, convert the client keystores into PKCS12 format and import them into your browser so you can pick which one to present to the server later on:
+
+```
+keytool -importkeystore -srckeystore client1.keystore -srcstoretype jks -destkeystore client1.keystore.pkcs12 -deststoretype pkcs12 -srcstorepass keystorepass -deststorepass keystorepass
+
+keytool -importkeystore -srckeystore client2.keystore -srcstoretype jks -destkeystore client2.keystore.pkcs12 -deststoretype pkcs12 -srcstorepass keystorepass -deststorepass keystorepass
```
Comment (Skyllarr) [inline client-cert-auth-without-roles/README.md:65]: @fjuma do we want to update the dynamic-certificates example to generate PKCS12 and not JKS ? Starting from Java 9, the default keystore used by Java is PKCS12 https://bugs.openjdk.org/browse/JDK-8044445 , if we do that we can skip this step

Comment (fjuma) [reply client-cert-auth-without-roles/README.md:65]: @Skyllarr yes that's a good idea. @PrarthonaPaul you can update that in this PR.

File: client-cert-auth-without-roles/simple-webapp/pom.xml (hunk 21 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/simple-webapp/pom.xml]: fixed!

File: client-cert-auth-without-roles/README.md (hunk 23 above)
Comment (fjuma) [reply client-cert-auth-without-roles/README.md:65]: Yes the type would need be updated there to PKCS12.

File: client-cert-auth-without-roles/README.md (hunk 22 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/README.md]: fixed!

**Review by cam-rod — APPROVED** [review]
Great work @PrarthonaPaul, everything works well!

**Review by Skyllarr — APPROVED** [review]
LGTM, thank you @PrarthonaPaul !

File: client-cert-auth-without-roles/README.md [hunk 24]
```diff
@@ -0,0 +1,145 @@
...
+Finally, copy the server.keystore.pkcs12 and server.truststore.pkcs12 files to your WildFly server instance:
+```
+
+cp $PATH_TO_ELYTRON_EXAMPLES/dynamic-certificates/target/server.* $WILDFLY_HOME/standalone/configuration
+
+```
+<a  name="serverConfiguration"></a>
+
+### Server Configuration
+The following set of instructions are going to be used to configure the Wildfly server in order to establish the mutual TLS connection. We will be deploying a simple web application under the `elytron-examples/client-cert-auth-without-roles/simple-webapp` folder.
```
Comment (fjuma) [inline client-cert-auth-without-roles/README.md:69]: s/Wildfly/WildFly

File: client-cert-auth-without-roles/simple-webapp/src/main/java/org/wildfly/security/examples/SecuredServlet.java [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ * A simple secured HTTP servlet.
+ *
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ */
+@WebServlet("/secured")
+// @ServletSecurity(httpMethodConstraints = { @HttpMethodConstraint(value = "GET") })
```
Comment (fjuma) [inline client-cert-auth-without-roles/simple-webapp/src/main/java/org/wildfly/security/examples/SecuredServlet.java:38]: This commented out line can be removed.

Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/simple-webapp/src/main/java/org/wildfly/security/examples/SecuredServlet.java]: FIxed!

File: client-cert-auth-without-roles/README.md (hunk 24 above)
Comment (PrarthonaPaul) [reply client-cert-auth-without-roles/README.md]: Fixed.
